application/src/main/java/org/opentripplanner/routing/algorithm/filterchain/filters/transit/RemoveTransitIfStreetOnlyIsBetter.java

```diff
@@ -15,10 +15,21 @@
  */
 public class RemoveTransitIfStreetOnlyIsBetter implements RemoveItineraryFlagger {
 
+  private static final Consumer<OptionalInt> IGNORE_SUBSCRIBER = i -> {};
+
   private final CostLinearFunction costLimitFunction;
+  private final OptionalInt streetOnlyCost;
+  private final Consumer<OptionalInt> streetOnlyCostSubscriber;
 
-  public RemoveTransitIfStreetOnlyIsBetter(CostLinearFunction costLimitFunction) {
+  public RemoveTransitIfStreetOnlyIsBetter(
+    CostLinearFunction costLimitFunction,
+    OptionalInt streetOnlyCost,
+    Consumer<OptionalInt> streetOnlyCostSubscriber
+  ) {
```

Comment:
Document when the streetOnlyCost is present and when the subscriber is present.

Reply:
I created documentation for this method